src/mlx/warnings/polyspace_checker.py

```diff
@@ -115,11 +116,10 @@ def return_check_limits(self):
         '''
         count = 0
         for checker in self.checkers:
-            print(
-                'Counted failures for family {!r} \'{}\': \'{}\''
-                .format(checker.family_value, checker.column_name, checker.check_value)
-            )
-            count += checker.return_check_limits()
+            padded_string = [f"{string:<30}" for string in [f"family {checker.family_value!r} ",
+                                                            f"{checker.column_name}: {checker.check_value} "]]
+            count += checker.return_check_limits("".join(padded_string))
+        print(f"Returning error code {count}.")
```

Comment:
If the exit code of the tool is 0, it is not an error code. 

```suggestion
        if count:
            print(f"Returning error code {count}.")
```